country-info.csv

```diff
@@ -164,6 +164,7 @@ Monash University,australasia,au
 Mälardalen University,europe,se
 NAIST,asia,jp
 NISER,asia,in
+NorthWest University,asia,cn

```

Comment:
The name is "Northwest" (no capital letter) in Wikipedia.
https://en.wikipedia.org/wiki/Northwest_University_(China)

Reply:
It also needs to be added to `institutions.csv`. This will cause the validation script to fail, but that's expected behavior.

Reply:
thank you very much. I fixed it and add Northwest University in institutions.csv